src/styles/index.scss

```diff
@@ -1,3 +1,126 @@
-body {
-  margin: 0;
-}
+ * {
+   margin: 0;
+   padding: 0;
+   box-sizing: border-box;
+ }
+
+ .card {
+   width: 200px;
+   height: 406px;
+   flex-shrink: 0;
+   border-radius: 5px;
+   border: 1px solid #f3f3f3;
+   background: #fff;
+   position: relative;
+
+   &__image {
+     margin-top: 32px;
+     margin-left: 19px;
+     width: 160px;
+     height: 134px;
+     background-image: url(../images/imac.jpeg);
+     background-size: cover;
+     background-position: center;
+   }
+
+   &__header,
+   &__product-code {
+     margin-left: 16px;
+   }
+
+   &__header {
+     margin-top: 36px;
+     margin-bottom: 4px;
+     width: 166px;
+     color: var(--main-accent, #060b35);
+     font-family: Roboto, sans-serif;
+     font-style: normal;
+     font-size: 12px;
+     font-weight: 500;
+     line-height: 18px;
+   }
+
+   &__product-code {
+    font-family: Roboto, sans-serif;
+     color: var(--secondary, #616070);
+     font-style: normal;
+     font-size: 10px;
+     font-weight: 400;
+     line-height: 14px;
+   }
+
+   &__reviews {
+     display: flex;
+     width: 53px;
+     height: 16px;
+     flex-direction: column;
+     justify-content: flex-end;
+     flex-shrink: 0;
+     color: var(--main-accent, #060b35);
+     text-align: right;
+     font-family: Roboto, sans-serif;
+     font-style: normal;
+     font-size: 10px;
+     font-weight: 400;
+   }
+
+   &__price-container {
+     margin-top: 24px;
+     margin-left: 16px;
+     display: flex;
+     justify-content: space-between;
+     align-items: flex-end;
+     width: 166px;
+   }
+
+   &__price-text,
+   &__price-value {
+     font-family: Roboto, sans-serif;
+   }
+
+   &__price-text {
+     color: var(--secondary, #616070);
+     font-size: 12px;
+     font-weight: 400;
+     line-height: 18px;
+     margin: 0;
+   }
+
+   &__price-value {
+     color: var(--main-accent, #060b35);
+     text-align: right;
+     font-size: 16px;
+     font-weight: 700;
+     line-height: 18px;
+   }
+
+   &__button {
+     margin-top: 16px;
+     margin-left: 16px;
+     display: block;
+     width: 166px;
+     height: 40px;
```

Comment:
it's a bad practice to hardcode height for elements with content inside. use padding

Reply:
isn't fixed. you still hardcoding width and height